docs/how-to/using-rocDecode-bitstream.rst

```diff
@@ -0,0 +1,76 @@
+.. meta::
+  :description: Using the rocDecode bitstream reader API

```

Comment:
Please change `API` to `APIs` here and in the other occurrences in this file because there are few APIs within the bitstream core functionality.